include/libopencm3/stm32/f4/rcc.h

```diff
@@ -665,6 +760,7 @@ enum rcc_periph_clken {
 	RCC_SPI6	= _REG_BIT(0x44, 21),/* F2xx, F3xx */
 	RCC_SAI1EN	= _REG_BIT(0x44, 22),/* F42x, F43x */
 	RCC_LTDC	= _REG_BIT(0x44, 26),/* F42x, F43x */
+	RCC_DSI		= _REG_BIT(0x44, 27),/* F4x9, F4x9 */
```

Comment:
You added some more ioports earlier, were there more to add here too? (port J and K)